tests/e2e/util/kubeUtils.go

```diff
@@ -106,8 +106,8 @@ func GetIngress(n string) (string, error) {
 func GetIngressPod(n string) (string, error) {
 	retry := Retrier{
 		BaseDelay: 5 * time.Second,
-		MaxDelay:  30 * time.Minute,
-		Retries:   10,
+		MaxDelay:  20 * time.Minute,
```

Comment:
Make it 5 minutes. We should not have to wait 20 mn for a test, that must be a mistake.